src/panels/MissionsPanel/TimeLine/Circle.tsx

```diff
@@ -0,0 +1,36 @@
+import { Tooltip } from '@mantine/core';
+import { ScaleTime } from 'd3';
+
+import { useOpenSpaceApi } from '@/api/hooks';
+import { useAppSelector } from '@/redux/hooks';
+
+import { jumpToTime } from '../util';
+
+interface CircleProps {
+  capture: string;
+  yScale: ScaleTime<number, number, never>;
+  marginLeft: number;
+  scale: number;
+}
+export function Circle({ capture, yScale, marginLeft, scale }: CircleProps) {
+  const now = useAppSelector((state) => state.time.timeCapped);
+  const luaApi = useOpenSpaceApi();
+  const circleRadius = 3;
+
+  return (
+    <Tooltip label={'Insturment Activity'}>
```

Comment:
```suggestion
    <Tooltip label={'Instrument Activity'}>
```

Reply:
Do we have data about what type of activity? This information seems pretty limited